### DISCUSSION TITLE
File is corrupt

### DISCUSSION THREAD

**Discussion by poteus** [ho-dev/HattrickOrganizer#1018] [Q&A] [answered]
I do not know if this a bug, but when I download data for one of my teams from HT, HO gives me a message "File is corrupt". I tried different things to the point to delete the database and restart fresh...
I still have the same error. 
How should I proceed from here?

**Comment by akasolace** [comment, marked as answer]
You should open a bug ticket and put a copy of your db as attachment ....

**Reply by poteus** [reply]
I was not sure if it was a bug. I will do that asap.

Thank you.
